packages/grafana-ui/src/components/Menu/MenuItem.tsx

```diff
@@ -225,7 +227,7 @@ const getStyles = (theme: GrafanaTheme2) => {
       justifyContent: 'center',
       padding: theme.spacing(0.5, 1.5),
       minHeight: theme.spacing(4),
-      borderRadius: theme.shape.radius.default,
+      borderRadius: getInternalRadius(theme, MENU_ITEM_PADDING),
```

Comment:
here i think you need to do `getInternalRadius(theme, MENU_ITEM_PADDING, { parentBorderWidth: 0 })`

Reply:
Same as the other one, I’ll push this up now!